amazon_price_per_item.user.js

```diff
@@ -641,17 +689,17 @@
 
           // Display results
           if (null !== foundPrice) {
-            if (null !== count) {
+            if (null !== countInDetail) {
               addEstimatedToElement(
                 foundPrice.pennies,
-                count,
+                countInDetail,
                 $addElementsTo,
                 1,
                 NEW_ITEM_DETAILS_CLASS
               );
             }
             $loading.text("");
-            $loading.append($("<div>" + foundPrice.price + "</div>"));
+            $loading.append($("<div class='my-price'>" + foundPrice.price + "</div>"));
```

Comment:
I don't see `my-price` being used anywhere?

Reply:
you're right, i don't know where it comes from !